dynamic-layers/phytec/recipes-bsp/u-boot/u-boot-imx_%.bbappend

```diff
@@ -23,3 +27,7 @@ do_configure:prepend() {
 do_configure:prepend:mx8mm-nxp-bsp() {
     cp -f ${WORKDIR}/omnect_env_phycore_imx8mm.h ${S}/include/configs/omnect_env_machine.h
 }
+
+do_deploy:append() {
+  echo "${PKGV}" > ${DEPLOYDIR}/bootloader_version
```

Comment:
does this work without `inherit deploy` or do we do that implicitly and i don't see it? what does the magic to move it from `${TMPDIR}/deploy` to `DEPLOY_DIR_IMAGE`?

Reply:
yes it works. This was my template for the DEPLOYDIR mechansim. -> https://github.com/phytec/meta-phytec/blob/master/recipes-bsp/u-boot/u-boot-imx_2022.04_2.2.1-phy3.bb#L39

I will try to find out how does it works without the inherit deploy class...

Reply:
The inherit deploy class is part of the this file https://github.com/phytec/meta-phytec/blob/d87bf33f5f8f6b638f9a889d7f6330183c9d1df9/recipes-bsp/u-boot/u-boot-imx_2022.04_2.2.1-phy3.bb#L4C1-L4C38